src/python/pcmdi/mean_climate_metrics_calculations.py

```diff
@@ -43,7 +43,7 @@ def compute_metrics(Var, dm, do):
 
     # SET CONDITIONAL ON INPUT VARIABLE
     if var == 'pr':
-        conv = 1.e5
+        conv = 86400. 
```

Comment:
@doutriaux1 @durack1   .... original units where kg/m2/s which are MKS... added conversion because a) mm/day are more commonly used for interpretation and b) kg/m2/s required many more significant digits than all vars